Essentia/UserCases/Login/Controllers/WelcomeViewController.swift

```diff
@@ -14,14 +14,7 @@ import EssDI
 import EssResources
 import RealmSwift
 
-class WelcomeViewController: BaseViewController, ImportAccountDelegate, SelectAccountDelegate {
-    
-    // MARK: - IBOutlet
-    @IBOutlet weak var restoreButton: UIButton!
-    @IBOutlet weak var title1Label: UILabel!
-    @IBOutlet weak var title2Label: UILabel!
-    @IBOutlet weak var enterButton: CenteredButton!
-    @IBOutlet weak var termsButton: UIButton!
+class WelcomeViewController: BaseTableAdapterController, ImportAccountDelegate, SelectAccountDelegate {
     
     // MARK: - Dependences
```

Comment:
Make all injection of dependencies  in this file under Dependencies mark

Reply:
done, f7d5855